tools/data_structs/pcap_buff.h

```diff
@@ -19,7 +19,9 @@ typedef enum {
     PKT_PADDING,
     PKT_RUNT,
     PKT_ERROR,
-    PKT_EOF
+    PKT_EOF,
+    PKT_OVER_SNAPLEN,
+    PKT_LEN_TRUNCATED // Don't like this name, any suggestions?
```

Comment:
Perhaps this might be called PKT_SNAPPED ? This case implies that the packet was snapped during capture. It's not a "error" but it may be useful to know when processing.